src/external-server/external-server.ts

```diff
@@ -169,7 +169,7 @@ export abstract class ExternalServer {
             return response.data;
         } catch (error) {
             const parent = error instanceof Error && error;
-            const msg = "Error in interaction with Starknet CLI proxy server";
+            const msg = `Error in interaction with Starknet CLI proxy server\n${error}`;
```

Comment:
We don't have to introduce a test for this, but were you able to test this locally and see what it looks like (how it's formatted)?

Reply:
Yes, I can post a picture here for ref purposes.
<img width="909" alt="image" src="https://user-images.githubusercontent.com/102951164/217208515-028e8f68-ab9f-4704-8f8e-1c2e3507c344.png">

Reply:
Ok, nice way of testing it!